Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+import os
+import sys
+import imp
+import codecs
+import subprocess
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+PLUGIN_CONF = '/etc/audisp/plugins.d/auoms.conf'
+OMSAGENT_CONF = '/etc/opt/microsoft/omsagent/conf/omsagent.d/auoms.conf'
+
+OMSAGENT_CONF_TEXT = """
+<source>
+  @type unix
+  path /var/opt/microsoft/auoms/sock/omsagent.sock
+</source>
+
+<match oms.api.audit>
+  type out_oms_api
+  log_level trace
+  buffer_chunk_limit 1m
+  buffer_type file
+  buffer_path /var/opt/microsoft/omsagent/state/out_oms_api_auoms.buffer
```

Comment:
please include * in the path, since it will create multiple buffers. out_oms_api_auoms*.buffer